### PR TITLE
add lldap to ldap

### PR DESCRIPTION
**Pull request by yater**

### CODE REVIEW

**Comment by yater** [comment]
Thank you for taking the time to work on a PR for Awesome-Sysadmin!

To ensure your PR is dealt with swiftly please check the following:

- [X] Your additions are [Free software](https://en.wikipedia.org/wiki/Free_software)
- [X] Software your are submitting is not your own, unless you have a healthy ecosystem with a few contributors (which aren't your sock puppet accounts).
- [X] Submit one item per pull request. This eases reviewing and speeds up inclusion.
- [X] Format your submission as follows, where `Demo` and `Clients` are optional.
  Do not add a duplicate `Source code` link if it is the same as the main link.
  Keep the short description under 80 characters and use [sentence case](https://en.wikipedia.org/wiki/Letter_case#Sentence_case)
  for it, even if the project's webpage or readme uses another capitalisation.
  `Demo` links should only be used for interactive demos, i.e. not video demonstrations.
  ``- [Name](http://homepage/) - Short description, under 250 characters, sentence case. ([Demo](http://url.to/demo), [Source Code](http://url.of/source/code), [Clients](https://url.to/list/of/related/clients-or-apps)) `License` `Language` ``
- [X] Additions are inserted preserving alphabetical order.
- [X] Additions are not already listed at [awesome-selfhosted](https://github.com/awesome-foss/awesome-sysadmin)
- [X] The `Language` tag is the main **server-side** requirement for the software. Don't include frameworks or specific dialects.
- [X] You have searched the repository for any relevant [issues](https://github.com/awesome-foss/awesome-sysadmin/issues) or [PRs](https://github.com/awesome-foss/awesome-sysadmin/pulls), including closed ones.
- [X] Any category you are creating has the minimum requirement of 3 items.
- [X] Any software project you are adding to the list is actively maintained.
- [X] The pull request title is informative, unlike "Update README.md".
  Suggested titles: "Add aaa to bbb" for adding software aaa to section bbb,
  "Remove aaa from bbb" for removing, "Fix license for aaa", etc.


--------------

### Why it is awesome

LDAP (with WUI) KISS style

**Review by nodiscc — CHANGES_REQUESTED** [review]
Hi @yater, since https://github.com/awesome-foss/awesome-sysadmin/pull/459 we have new contribution guidelines in place, could you please take some time to answer the following questions as best you can? Thanks

- **Why is it awesome?**
- **Have you used it? For how long?**
- **Is this in a personal or professional setup?**
- **How many devices/users/services/... do you manage with it?**
- **Biggest pros/cons compared to other solutions?**

**Comment by pixelrazor** [comment]
To add my own experience:


 -   Why is it awesome? - Incredibly easy to set up and use. Very lightweight. It allowed me to have absolutely no ldap knowledge and still have ldap to centralize auth in my home setup.
 -   Have you used it? For how long? Yes! i'd say since around november 2022 - so ~4 months?
 -   Is this in a personal or professional setup? Personal
 -   How many devices/users/services/... do you manage with it? I have 6 users, and it's integrated with 3 of my local services
 -   Biggest pros/cons compared to other solutions? Pros - incredibly easy to use, simple, intuitive UI, Tons of thorough examples for integrating with other services (i was able to use them to configure services  and maintain zero knowledge of ldap itself), and light on resources. Cons - As of current, the things it supports is pretty minimal. The attributes/fields on things are limited, with no was to add more (so no adding ssh keys or anything - but AFAIK that will be in it hopefully sometime this year?). 

LLDAP is awesome, through and through. I tell all my friends to use it for their servers, and i always try to talk one of my friends into using it in his professional environment where they use the most minimal openLDAP to do local and integration tests

File: README.md [hunk 1]
```diff
@@ -359,6 +359,7 @@ _See also: [IT Asset Management]([Ralph](#it-asset-management))_
 * [OpenDS](https://opends.java.net/) - Another directory server written in Java.
 * [OpenLDAP](http://www.OpenLDAP.org/) - Developed by the OpenLDAP Project.
 * [FreeIPA](http://www.freeipa.org/) - Based on 389-DS. Includes Kerberos, DNS, as well as host based access control.
+* [lldap](https://github.com/nitnelave/lldap) - Light (simplified) LDAP implementation written in Rust and a web interface with Node.js ([Source Code](https://github.com/nitnelave/lldap)) `GPL-3.0` `Rust`/`JavaScript`.
```
Comment (nitnelave) [inline README.md:362]: Just a comment from the author: There's no JS (unless you count the 5 lines to bootstrap the WASM frontend). The web interface is _not_ in Node.js but in Rust compiled to WALM (with the Yew framework, if it's relevant).

In the one-line blurb, I would maybe talk more about the main goals: easy to deploy and to use (but less featureful than other LDAP servers).

**Comment by lucat1** [comment]
- **Why is it awesome?** Great community/author support, super slim and easy to configure, especially compare to full fledged LDAP servers. Has a nice builtin web ui. Feature-wise, supports basically everything one may need to do authentication with LDAP (the only missing thing, SSH keys support, is hopefully coming).
- **Have you used it? For how long?** I've been using it for around 4 months. I've also had the chance to contribute to the project in the meantime. The maintainer is super friendly and helpful.
- **Is this in a personal or professional setup?** It's a semi-professional setting, we hold a database of university students that use services we offer. There is demand for stability and LLDAP has always delivered.
- **How many devices/users/services/... do you manage with it?** We have around 20 users, connected to 5 services. When a feature was missing to support a service we needed, the maintainer implemented it  swiftly.
- **Biggest pros/cons compared to other solutions?** KISS, light on resources and stores password in a way more secure manner, compared to others. Ease of setup and maintainability are also huge reasons why we chose it.

**Review by nodiscc — APPROVED** [review]

**Pull request merged**